src/main/resources/config/config.yml

```diff
@@ -22,3 +22,5 @@ compose:
     active: false
   ComposableFunctionName:
     active: true
+  UnnecessaryLayoutWrapper:
+    active: true
```

Comment:
This also needs to be added to the sample block in the `README.md`, some people will be copying rules from here (config.yml doesn't work for older detekt versions)

Reply:
Secondly, this rule must be added to the [rule set provider](https://github.com/appKODE/detekt-rules-compose/blob/master/src/main/kotlin/ru/kode/detekt/rule/compose/ComposeRuleSetProvider.kt).